packages/plugins/cloud/admin/src/pages/HomePage/index.js

```diff
@@ -0,0 +1,166 @@
+/*
+ *
+ * HomePage
+ *
+ */
+
+import React, { useState, useEffect } from 'react';
+// import PropTypes from 'prop-types';
+
+import { LoadingIndicatorPage } from '@strapi/helper-plugin';
+import { verifyIfProjectIsVersionOnGit } from '../../utils/api';
+
+import { Box, GridLayout, Flex, Typography, Link } from '@strapi/design-system';
+
+import styled from 'styled-components';
+
+import GithubBox from '../../components/Github';
+import CloudBox from '../../components/Cloud';
+
+import cornerOrnamentPath from './assets/corner-ornament.svg';
+import rightSideCloudPath from './assets/right-side-cloud.png';
+import leftSideCloudPath from './assets/left-side-cloud.png';
+
+import { useIntl } from 'react-intl';
+import { getTrad } from '../../utils';
+
+const LogoContainer = styled(Box)`
+  position: absolute;
+  top: 0;
+  right: 0;
+
+  img {
+    width: ${150 / 16}rem;
+  }
+`;
+
+const RightSideCloudContainer = styled(Box)`
+  position: absolute;
+  top: 400px;
+  right: 0;
+
+  img {
+    width: ${150 / 16}rem;
+  }
+`;
+
+const LeftSideCloudContainer = styled(Box)`
+  position: absolute;
+  top: 150px;
+  left: 220px;
+
+  img {
+    width: ${150 / 16}rem;
+  }
+`;
+
+const HomePage = () => {
+  const { formatMessage } = useIntl();
+
+  const [isLoading, setIsLoading] = useState(true);
+  const [isVersionedOnGit, setIsVersionedOnGit] = useState(false);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const res = await verifyIfProjectIsVersionOnGit();
+        setIsVersionedOnGit(res || false);
+      } catch (error) {
+        console.log(error);
+      }
+    };
+    fetchData().then(() => {
+      setIsLoading(false);
+    });
+  }, []);
```

Comment:
This is a bad way to fetch data. You should use react-query like we do in the rest of the codebase please.